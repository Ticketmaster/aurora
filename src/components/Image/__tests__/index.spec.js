import React from "react";
import renderer from "react-test-renderer";

import CircleImage from "../Circle";

describe("<CircleImage />", () => {
  it("renders circle image correctly", () => {
    const component = renderer.create(
      <CircleImage
        size={40}
        src="https://placekitten.com/g/512/288"
        alt="circleImage"
      />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
