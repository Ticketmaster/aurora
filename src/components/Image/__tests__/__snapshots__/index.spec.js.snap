// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ThumbnailCircle /> renders ThumbnailCircle image correctly 1`] = `
<div
  className="lovFLS"
  size={40}
>
  <img
    alt="thumbnailCircle"
    className="leiZSW"
    src="https://placekitten.com/g/512/288"
  />
</div>
`;
